### PR TITLE
Quick Sort

### PR DESCRIPTION
**Pull request by vanshita25**
One of the ways of sorting

### CODE REVIEW

File: quicksort.c [hunk 1]
```diff
@@ -0,0 +1,54 @@
+#include<stdio.h>
+void QuickSort(int [],int,int);
+int Partition(int [],int,int);
+void main()
+{
+    int a[10]={4,3,6,8,1,10,15,20,34,12};
```
Comment (Bhavnaharitsa) [inline quicksort.c]: no single letter variable

File: quicksort.c [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    int high=9;
+    QuickSort(a,0,9);
+    printf("\nSorted array is :");
+	for(i=0;i<9;i++)
+		printf("%d ",a[i]);
+
+}
+void QuickSort(int a[],int low,int high)
+{
+    int k;
```
Comment (Bhavnaharitsa) [inline quicksort.c]: same

File: quicksort.c [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        if(i<j)
+        {
+           t=a[i];
+           a[i]=a[j];
+           a[j]=t;
+        }
+    }while(i<j);
+
+    t1=a[low];
+    a[low]=a[j];
```
Comment (Bhavnaharitsa) [inline quicksort.c]: same

File: expressionvalidity.c [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+};
+struct stack s;
+int Intialize(void)
+{
+    s.top=-1;
+}
+ int StackTop()
+ {
+     int x;
+     x=s.Item[S.top];
```
Comment (Bhavnaharitsa) [inline expressionvalidity.c]: What is X?

File: expressionvalidity.c [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+}
+int Pop()
+{
+    int x;
+    if(Empty(s))
+        printf("Stack Underflow");
+    else
+        x=s.Item[s.top];
+        s.top--;
+        return x;
```
Comment (Bhavnaharitsa) [inline expressionvalidity.c]: What is s?

File: expressionvalidity.c [hunk 6]
```diff
@@ -0,0 +1,95 @@
+#include<stdio.h>
+#define size 10
+#define TRUE 1
+#define FALSE 0
```
Comment (Bhavnaharitsa) [inline expressionvalidity.c]: Sample input output description

File: expressionvalidity.c [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+        return FALSE;
+}
+void Push(int x)
+{
+    if(s.top==size-1)
+        printf("Stack Overerflow");
+    else
+        s.top=s.top+1;
+        s.Item[s.top]=x;
+}
```
Comment (Bhavnaharitsa) [inline expressionvalidity.c:37]: Why push is void and pop int

**Comment by vanshita25** [comment]
please review the changes of PR #74

File: expressionvalidity.c (hunk 4 above)
Comment (vanshita25) [reply expressionvalidity.c]: do i need to give the variable description or has to change the variables ?

**Comment by vanshita25** [comment]
please review the updated expression validity program

**Pull request closed without merging**